dashboard/config/routes.rb

```diff
@@ -336,7 +336,14 @@
     end
   end
 
-  resources :programming_classes, only: [:new, :create, :edit, :update, :show]
+  resources :programming_classes, only: [:new, :create, :edit, :update, :show, :destroy] do
+    collection do
+      get :get_filtered_expressions
```

Comment:
nit: expressions isn't the right term here, maybe `get_filtered_classes`? Also feel free to decide it's not worth it or do as a follow up

Reply:
Yes I'd love to change this... is there a name that would suit both? The simplicity of this implementation hinges on it being called the same thing in both controllers

Reply:
maybe just get_filtered_results?

Reply:
Ahhh that makes sense. I think `get_filtered_results` makes sense as the full urls have the model name in them

Reply:
done!